docs/pages/zero-trust-access/infrastructure-as-code/managing-resources/user-and-role.mdx

```diff
@@ -1,5 +1,6 @@
 ---
 title: Managing Users And Roles With IaC
+sidebar_label: Users
```

Comment:
nit, maybe "Users And Roles"? We might lose some information by dropping the reference to roles, maybe?


```suggestion
sidebar_label: Users And Roles
```

Reply:
My mistake. Thanks for catching!